app.js

```diff
@@ -69,10 +76,16 @@ function photoClick(event){
 
   if (clicksRemaining > 0){
     renderPhotos();
+
   } else {
     renderChart();
   }
 }
+try {
+  localStorage.photos = JSON.stringify(photos);
+} catch (error) {
+  console.log('something went wrong', error);
```

Comment:
your error messages should tell you what they're related to without giving people an idea of what they can do to break it. (i.e. "failed to save to local storage")